src/cc65/declare.c

```diff
@@ -2137,6 +2137,7 @@ static unsigned ParseStructInit (Type* T, int AllowFlexibleMembers)
     ConsumeRCurly ();
 
     /* If we have data from a bit-field left, output it now */
+    CHECK(SI.ValBits < CHAR_BITS);
```

Comment:
Style fix: a space between the macro-name and the parenthesis.

Reply:
Done.